packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -185,30 +142,12 @@ const Fieldset = ({
     );
   }
```

Comment:
Since this is never re-assigned we could clean it up by moving it directly to where it is rendered how it is:

```tsx
{onEdit && (
  <ActionButton {....props} />
)}
```

Same with the remove button.